demo/manufacturer/manufacturer.env

```diff
@@ -6,4 +6,8 @@ manufacturer_database_port=8049
 catalina_home=./target/tomcat
 manufacturer_keystore_password=MfgKs@3er
 manufacturer_api_user=apiUser
-manufacturer_api_password=MfgApiPass123
\ No newline at end of file
+manufacturer_api_password=MfgApiPass123
```

Comment:
The OnDie cert cache is provided for protocol samples only. We are running smoketest using demo folder which uses component-sample binaries. The component sample manufacturer reads the ondie cache path from .env file. There is no default value provided in the .env file, so manufacturer docker is running into "OnDie Initialization Error"

Also autoupdate property should be set to true by default because we are running component-samples via docker and the .cer and .crl files needs to be downloaded everytime the docker container is created.

Reply:
Changing initialization to support case where no OnDie configuration is specified in cases of backwards compatibility and when the user does not need to support OnDie.

Reply:
done